### PR TITLE
Enable multiple doc trees per app

### PR DESCRIPTION
**Pull request by buehner**
- Rename RBMA to document
- Add a document listing for each application
- New docs can be created for an application

### CODE REVIEW

File: app/view/panel/document/DocumentTreeController.js [hunk 1]
```diff
@@ -23,25 +24,26 @@ Ext.define('MoMo.client.view.panel.rbma.RbmaTreeController', {
     },
 
     /**
-     * Set store for the RBMA document tree
+     * Set store for the document tree
      */
-    setRbmaStoreAndRootNode: function() {
+    setDocumentStoreAndRootNode: function() {
         var me = this,
```
Comment (KaiVolland) [inline app/view/panel/document/DocumentTreeController.js]: MINOR: Would be cleaner if you won't mix comma seperated variable declaration with the semicolon way you did before.

**Review by KaiVolland — APPROVED** [review]
Looks good to me. The use of viewmodels don't seem to be so important. Mostly you'll profit from them later when you didn't expected it...

If you'd like to know how you could have done this with viewmodels you might want to look at:

https://www.sencha.com/forum/showthread.php?284233-Passing-data-from-controller-to-viewModel&p=1039698&viewfull=1#post1039698

and 

https://www.sencha.com/forum/showthread.php?313403-Passing-parameter-from-View-to-Viewmodel&p=1141344&viewfull=1#post1141344

File: app/view/panel/document/DocumentTreeController.js (hunk 1 above)
Comment (buehner) [reply app/view/panel/document/DocumentTreeController.js]: Fixed it...

**Comment by buehner** [comment]
Thx for review and the hints. I'll merge now...

**Pull request merged**